Core/src/RequestHandler.php

```diff
@@ -135,8 +135,19 @@ public function sendRequest(
      * @param $clientClass The client class whose object we need.
      * @return mixed
      */
-    private function getClientObject(string $clientClass)
+    public function getClientObject(string $clientClass)
     {
         return $this->clients[$clientClass] ?? null;
     }
+
+    /**
+     * Helper function to add a client object to the $clients array.
+     *
+     * @param $clientName The class name of the client.
+     * @param $clientObject The client object to be added.
+     */
+    public function addClientObject(string $clientName, mixed $clientObject)
```

Comment:
These methods seem hacky and they could have unexpected results. This points to a flaw in the design.